### ISSUE TITLE
When editing a order on wp-admin we need to be able to populate additional address fields

### ISSUE THREAD

**Issue by webdados** [woothemes/woocommerce#9444]
Hi there,

I've written a small plugin to add the Portuguese VAT number to the Billing Address on Checkout, My Account -> Billing Address, Order emails, Thank You page and, of course, the order admin panel.
This field is not only saved on the order details but also on the user details/address.

The shop owner can edit any billing address fields including my custom field but when loading the client details from an existent client, it's not possible to populate my field.

I've already added a filter to 'woocommerce_found_customer_details' so that the ajax call with action 'woocommerce_get_customer_details' and type_to_load 'billing' returns also my custom field, but then javascript code (meta-boxes-order.php, lines 147 to 159) only takes care of core fields.

Bottom line: if a developer is able to add custom fields to the order address admin panel, it should be also able to populate them via ajax.

**Comment by claudiosmweb** [comment]
Create a custom ajax action for it.

**Comment by mikejolley** [comment]
@claudiosmweb I can improve this

**Comment by webdados** [comment]
@mikejolley thanks
